PyInstaller/compat.py

```diff
@@ -281,7 +282,12 @@ def unsetenv(name):
 # Exec commands in subprocesses.
 
 
-def exec_command(*cmdargs: str, encoding: str = None, raise_enoent: bool = None, **kwargs):
+def exec_command(
+    *cmdargs: str,
+    encoding: str | None = None,
+    raise_enoent: bool | None = None,
+    **kwargs: int | bool | Iterable | None
```

Comment:
Does it make sense to explicitly annotate `**kwargs`? (Same goes for other functions)

Reply:
Annotating `**kwargs` specifies the potentially valid types for the dict's values (in this case it's equivalent to `kwargs: dict[str, int | bool | Iterable | None]`. As for how I came up with those specific types: I just followed the kwargs chain down.

Similarly, for `*cmdargs: str` -> `cmdargs: list[str]`